gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/importing/ImportBundleTask.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2018 CA. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayexport.tasks.importing;
+
+import com.ca.apim.gateway.cagatewayexport.config.GatewayConnectionProperties;
+import com.ca.apim.gateway.cagatewayexport.util.http.GatewayClient;
+import com.ca.apim.gateway.cagatewayexport.util.http.GatewayClientException;
+import org.apache.http.entity.StringEntity;
+import org.gradle.api.DefaultTask;
+import org.gradle.api.file.RegularFileProperty;
+import org.gradle.api.tasks.Nested;
+import org.gradle.api.tasks.TaskAction;
+
+import java.io.File;
+import java.io.IOException;
+
+import static com.ca.apim.gateway.cagatewayexport.util.injection.ExportPluginModule.getInstance;
+import static java.nio.charset.Charset.defaultCharset;
+import static org.apache.commons.io.FileUtils.readFileToString;
+import static org.apache.http.HttpHeaders.CONTENT_TYPE;
+import static org.apache.http.client.methods.HttpPut.METHOD_NAME;
+import static org.apache.http.client.methods.RequestBuilder.create;
+
+/**
+ * Task to import a bundle into an existing running gateway.
+ */
+public class ImportBundleTask extends DefaultTask {
+
+    private GatewayClient gatewayClient;
+
+    //Inputs
+    private GatewayConnectionProperties gatewayConnectionProperties;
+    private RegularFileProperty importFile;
+
+    public ImportBundleTask() {
+        this.gatewayClient = getInstance(GatewayClient.class);
+        gatewayConnectionProperties = new GatewayConnectionProperties(getProject());
+        importFile = newInputFile();
+
+        // makes it so that the export is always run
+        getOutputs().upToDateWhen(t -> false);
```

Comment:
is this line needed?

Reply:
yes, otherwise if the file does not change, gradle assumes that does not have to re-run the task
doing this, we always run the task when it's specified.